aspnetcore/mvc/views/tag-helpers/th-components/samples/RazorPagesSample/Pages/Contact.cshtml.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using Microsoft.AspNetCore.Mvc.RazorPages;
+
+namespace RazorPagesSample.Pages
+{
+    public class ContactModel : PageModel
+    {
+        public string Message { get; set; }
+
+        public bool IsWeekend
+        {
+            get
+            {
+                var dayOfWeek = DateTime.Now.DayOfWeek;
+
+                return dayOfWeek == DayOfWeek.Saturday ||
```

Comment:
**_OMG! ... Major coding error here!_** It should be .....

```csharp
return dayOfWeek == DayOfWeek.Friday ||
       dayOfWeek == DayOfWeek.Saturday ||
       dayOfWeek == DayOfWeek.Sunday;
```

Reply:
Total novice behind the keyboard here.

Reply:
lol ... *touche!* ... It doesn't take into account those great Saturday morning sessions tho. I really think I do my best work on Saturday morning. It's quiet here. There's no rushing out to the :muscle: gym :muscle: in the morning. Plenty of good ☕️ on tap.